test/lib/collectionFactory.test.js

```diff
@@ -90,6 +90,34 @@ describe('Collection Factory', () => {
                 });
             });
         });
+
+        it('should create a Collection without pipelineIds', () => {
+            const dataWithoutPipelineIds = Object.assign({}, collectionData);
+
+            delete dataWithoutPipelineIds.pipelineIds;
+            datastore.save.resolves(dataWithoutPipelineIds);
```

Comment:
Whoops! You're right, I messed up on that. But the main part of the test is actually [this line](https://github.com/screwdriver-cd/models/pull/184/files#diff-1804da3236cbfdf1b4a8055e43ebb888R111). I'll change the test right now